### ISSUE TITLE
Other ways to help

### ISSUE THREAD

**Issue by cesasol** [oyranos-cms/oyranos#23]
I'm a graphic/web designer and recently moved to OpenSUSE Leap, and wanted to help by doing something that is not codding. 
I guess packaging this with OBS will be a nice start.

**Comment by beku** [comment]
Yes, would be _very_ helpful. There are many smaller packages involved. Most things are packaged, but definitely need some love like Synnefo and Qt5. Please see the following OBS repos:
* https://build.opensuse.org/package/show?project=home%3Abekun%3Adevel&package=oyranos
* https://build.opensuse.org/package/show?project=multimedia%3Acolor_management&package=oyranos
* https://bugzilla.opensuse.org/show_bug.cgi?id=958071

Btw. sorry for the long delay in answering.
